docs/cmd/kumactl/kumactl_config_control-planes_add.md

```diff
@@ -17,6 +17,7 @@ kumactl config control-planes add [flags]
       --auth-conf stringToString   authentication configuration for defined authentication type format key=value (default [])
       --auth-type string           authentication type (for example: "tokens")
       --ca-cert-file string        path to the certificate authority which will be used to verify the Control Plane certificate (kumactl stores only a reference to this file)
+      --check-timeout duration     the timeout of the first api call to check connectivity to the api server (probably never needs to be updated) (default 5s)
```

Comment:
should we do `--api-server-timeout` instead? Right now there is no way to configure the timeout for requests to API server

Reply:
Arg you caught me on the more complex change :)
Yes it'd make sense I'll do this